internals/secrethub/run.go

```diff
@@ -174,7 +227,7 @@ func (cmd *RunCommand) Run() error {
 	maskStrings := make([][]byte, len(secrets))
 	i := 0
 	for _, val := range secrets {
-		maskStrings[i] = val
+		maskStrings[i] = []byte(val)
```

Comment:
This does not contain the values from `.env` files.

Reply:
I've opened #115 to fix this.